ckanext/depositar_theme/helpers.py

```diff
@@ -67,3 +72,43 @@ def send_reg_link(user):
     subject = subject.split('\n')[0]
 
     mail_user(user, subject, body)
+
+@core_helper
+def get_format_count():
+    results = logic.get_action('package_search')({}, {})['results']
+    format_list = []
+    for res in results:
+        resourceList = res['resources']
+        for resource in resourceList:
```

Comment:
變數命名風格應一致 ([PEP 8](https://www.python.org/dev/peps/pep-0008/#id34))

```suggestion
        resource_list = res['resources']
        for resource in resource_list:
```